taxi/views.py

```diff
@@ -36,6 +44,24 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_queryset(self):
+        name = self.request.GET.get("name")
+        queryset = Manufacturer.objects.all()
+
+        if name:
+            return queryset.filter(name__icontains=name)
+        return queryset
+
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(ManufacturerListView, self).get_context_data()
```

Comment:
no need to pass class name to super() explicitly; 
**kwargs should be passed to get_context_data